scripts/migration/renamings.json5

```diff
@@ -1478,5 +1478,9 @@
         },
       },
     },
+    {
+      oldName: 'Blockly.Icon',
+      newName: 'Blockly.icons.Icon',
+    }
```

Comment:
@cpcallen I think this is correct, b/c the export didn't get renamed, but if you could verify that would be appreciated!

Reply:
There doesn't seem to be a goog.module ID for this module, so strictly speaking there's not a correct way to represent this in the renamings file as it is presently structured, but this will have the desired effect.